buildlib/az-distro-release.yml

```diff
@@ -31,6 +31,19 @@ jobs:
         ubuntu22_cuda11:
           build_container: ubuntu22_cuda11
           artifact_name: $(POSTFIX)-ubuntu22.04-mofed5-cuda11.tar.bz2
+        centos7_cuda12:
```

Comment:
CentOS 8/9 images with CUDA 12 are not available:
https://hub.docker.com/r/nvidia/cuda/tags?page=1&name=12.0.0-devel-centos
Probably because CentOS 8 is EOL.

Reply:
@yosefe do we need UCX community release for rhel/centos 8/9?

Reply:
> CentOS 8/9 images with CUDA 12 are not available: https://hub.docker.com/r/nvidia/cuda/tags?page=1&name=12.0.0-devel-centos Probably because CentOS 8 is EOL.

Yes, but I meant ubi8/9 based images applicable for rhel/centos

Reply:
UBI is to be added by another PR.